app/src/canary-file.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This is WIP to add a test as it will remain in the codebase to check if its present